src/Microsoft.ML.Data/Transforms/ExtensionsCatalog.cs

```diff
@@ -107,8 +107,30 @@ public static ColumnSelectingEstimator DropColumns(this TransformsCatalog catalo
         /// </example>
         public static ColumnSelectingEstimator SelectColumns(this TransformsCatalog catalog,
             string[] keepColumns,
-            bool keepHidden = ColumnSelectingTransformer.Defaults.KeepHidden)
+            bool keepHidden = ColumnSelectingEstimator.Defaults.KeepHidden)
+            => new ColumnSelectingEstimator(CatalogUtils.GetEnvironment(catalog),
+                keepColumns, null, keepHidden, ColumnSelectingEstimator.Defaults.IgnoreMissing);
+
+        /// <summary>
+        /// ColumnSelectingEstimator is used to select a list of columns that user wants to keep from a given input.
+        /// </summary>
+        /// <remarks>
+        /// <format type="text/markdown">
+        /// When instantiated through this constructor, <see cref="ColumnSelectingEstimator"/> operates on the schema of an input IDataView
```

Comment:
><see cref="ColumnSelectingEstimator"/> [](start = 56, length = 38)

i don't think <see cref will  work inside markdown. 

I think we have to fully specify the assembly in this style:

<xref:Microsoft.ML.Transforms.ColumnSelectingEstimator>

Reply:
Ok good to know will fix in this file

---
In reply to: [254369733](https://github.com/dotnet/machinelearning/pull/2367#discussion_r254369733) [](ancestors = 254369733)